### DISCUSSION TITLE
AST selector to match arguments, but not callee?

### DISCUSSION THREAD

**Discussion by iliubinskii** [eslint/eslint#15771] [Q&A] [answered]
Sample code:
```ts
console.log(1, 2);
```
I want to match only "1" and "2".
I am trying "CallExpression > *", but it also matches "console.log".
How can I match only "console.log" (i.e. only callee) or only "1" and "2" (i.e. only arguments)?
Thx for the help.

**Comment by mdjermanovic** [comment, marked as answer]
`CallExpression > *.callee` matches only callee, `CallExpression > *.arguments` matches only arguments.

[ESLint Demo link](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgbm8tcmVzdHJpY3RlZC1zeW50YXg6IFtcImVycm9yXCIsIFxuICAgIHtcbiAgICAgICAgc2VsZWN0b3I6IFwiQ2FsbEV4cHJlc3Npb24gPiAqLmNhbGxlZVwiLFxuICAgICAgICBtZXNzYWdlOiBcIlRoaXMgbWF0Y2hlcyBjYWxsZWVcIiAgICAgICAgXG4gICAgfSxcbiAgICB7XG4gICAgICAgIHNlbGVjdG9yOiBcIkNhbGxFeHByZXNzaW9uID4gKi5hcmd1bWVudHNcIixcbiAgICAgICAgbWVzc2FnZTogXCJUaGlzIG1hdGNoZXMgYXJndW1lbnRzXCIgICAgICAgIFxuICAgIH1cbl0qL1xuXG5jb25zb2xlLmxvZygxLCAyKTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOiJsYXRlc3QiLCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19)

**Comment by iliubinskii** [comment]
Thx. That works.
